### ISSUE TITLE
Option to generate HTML report in addition to showing errors in terminal

### ISSUE THREAD

**Issue by 4leem** [MoOx/eslint-loader#89]
Is it possible to generate HTML output of the eslint errors and warnings ? Something similar to [eslint-html-reporter](https://github.com/ratherblue/eslint-html-reporter) which works well with the gulp.

**Comment by MoOx** [comment]
That seems to be a good idea, I like it. That should be an option so people can enable or disable it.
PR welcome !

**Comment by vidhill** [comment]
I was trying to do similar, to write out eslint errors in 'checkstyle' format for display on Jenkins,
It's quite a simple solution, so mightn't be very robust


```
options: {
                eslint: {
                    formatter: function(results){
                        var formatters =  {
                            checkstyle: require('eslint/lib/formatters/checkstyle'),
                            stylish: require('eslint/lib/formatters/stylish'),
                        };
                        // parse the lint result in checkstyle format and write out a file
                        var formatedRes = formatters.checkstyle(results);
                        fs.writeFile( resolve(__dirname, 'dist') + '/result.txt', formatedRes);

                        // return the default output into the console
                        return formatters.stylish(results);
                    }
                }
            }
```

**Comment by MoOx** [comment]
Closed by #118
